src/main/java/common/tileentities/GTMTE_LapotronicSuperCapacitor.java

```diff
@@ -908,7 +910,10 @@ public String[] getInfoData() {
         final ArrayList<String> ll = new ArrayList<>();
         ll.add(EnumChatFormatting.YELLOW + "Operational Data:" + EnumChatFormatting.RESET);
         ll.add("Used Capacity: " + nf.format(stored) + "EU");
+        ll.add("Used Capacity: " + toPercentageFrom(stored, capacity, 6) + "%");
+        ll.add("Used Capacity: " + toStandardForm(stored) + "EU");
```

Comment:
This is an odd order to display the info. It would be preferable if it was:

> Used capacity: 123,456,789 EU
> Used capacity: 1.23x10^9 EU
> Used capacity: 12.3%

rather than having the percentage between the two absolute readouts.

Or, IMO, even better, something like:

> EU Stored: 123,456,789 (1.23x10^9)
> Used capacity: 12.3%

As it is right now, it might not even be immediately obvious that the three readouts all correspond to the same value.

Reply:
did it
![image](https://github.com/GTNewHorizons/KekzTech/assets/17906427/4002d397-0e29-4857-9983-a3f24feeef1e)

Reply:
make the total capacity in parentheses too?

Reply:
these lines are already long enough, so those who use the OС may be dissatisfied.

Reply:
Either put both on the same line, or both separately. Any way is good, just be consistent.